src/screens/simulation/Simulation.js

```diff
@@ -121,7 +121,9 @@ const Simulation = () => {
             <Grid container>
                 <Grid item xs={5}>
                     {values ? (
-                        <Chart values={values} startDate={startDate} />
+                        <div style={{ position: 'fixed', width: 800 }}>
```

Comment:
Look at Chart.js, its width vary with the screen size
You can copy/paste the code for the width ;)